cli/github.go

```diff
@@ -86,3 +87,27 @@ func newGithubTrendingCmd() cli.Command {
 	}
 
 }
+
+func newGithubRepositoryCmd() cli.Command {
+	return cli.Command{
+		Name:    "repository",
+		Aliases: []string{"r"},
+		Usage:   "Open the repository page",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "org",
+				Usage: "Organization to open",
```

Comment:
`org` flag include some `github_user_id` like `KeisukeYamashita` . 🤕 
So please comment if you have any idea.

Reply:
Let's take a look at what Github API defines the API.

For listing repos they have two APIs.

```
 /users/:username/repos
```

```
/orgs/:org/repos
```

So, from this point of view, I suggest you to implement two flags `-org` and `-user`.
If the user specifies two flags in a single command, we can return an error.

Any thoughts?

Reply:
@KeisukeYamashita 
I just add flag of`users` !